src/wanderer.ts

```diff
@@ -46,165 +52,200 @@ export function digitizedMonstersRemaining(): number {
   );
 }
 
-interface ZonePotion {
+interface ZoneUnlocker {
   zone: string;
-  effect: Effect;
-  potion: Item;
+  available: () => boolean;
+  unlocker: Item;
+  noInv?: boolean;
+}
+
+function airportAvailable(element: "spooky" | "stench" | "hot" | "cold" | "sleaze"): boolean {
```

Comment:
I instinctively want this to use a $element, but that is probably wrong of me

Reply:
$element includes lime and supercold, which are not valid airports. I can look and see if there is a way to exclude only specific elements, or modify libram to classify those elements different or provide a list to them. PrimaryElement maybe?

Reply:
yeah that sucks then